CHANGES.md

```diff
@@ -0,0 +1,6303 @@
+# GAP - Changes from Earlier Versions
```

Comment:
Do we want this "Changes from Earlier Versions" title, or just say "Changes history" ?